src/fetch/transfer_file.py

```diff
@@ -362,31 +364,87 @@ def __str__(self) -> str:
         )
 
 
+def map_reward(
+    amount: float,
+    risk_free: bool,
+    batch_contains_unsafe_liquidity: bool,
+) -> float:
+    """
+    Converts orderbook rewards based on additional knowledge of "risk_free" transactions
+    """
+    if amount > 0 and risk_free and not batch_contains_unsafe_liquidity:
+        # Risk Free Orders that are not liquidity orders get 37 COW tokens.
+        return 37.0
+    return amount
+
+
+def liquidity_order_batches(order_df: DataFrame) -> set[str]:
+    """
+    Fetches the set of transaction hashes containing a
+    liquidity order as the unique tx_hash where reward `amount = 0`
```

Comment:
"containgin a liquidity order as the unique tx_hash" I don't understand this

Reply:
I will update this to be more clear.